.github/workflows/compare-layouts.yml

```
@@ -0,0 +1,240 @@
+      - name: Restore the layout files from the artifact
+        if: ${{ github.event.workflow_run.conclusion == 'success' }}
+        uses: dawidd6/action-download-artifact@v6
+        with:
+          name: storage-layouts-${{ github.event.workflow_run.head_commit.id }}
+          run_id: ${{ github.event.workflow_run.id }}
+      - name: Extract the restored layouts
+        if: ${{ github.event.workflow_run.conclusion == 'success' }}
+        run: unzip storage-layouts.zip
+      - name: Set up Node.js
+        if: ${{ github.event.workflow_run.conclusion == 'success' }}
+        uses: actions/setup-node@v2
```

actionlint (1.7.4)

the runner of "actions/setup-node@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)